patterns/firewalls

```diff
@@ -79,8 +93,22 @@ CISCOFW602303_602304 %{WORD:protocol}: An %{CISCO_DIRECTION:direction} %{GREEDYD
 CISCOFW710001_710002_710003_710005_710006 %{WORD:protocol} (?:request|access) %{CISCO_ACTION:action} from %{IP:src_ip}/%{INT:src_port} to %{DATA:dst_interface}:%{IP:dst_ip}/%{INT:dst_port}
 # ASA-6-713172
 CISCOFW713172 Group = %{GREEDYDATA:group}, IP = %{IP:src_ip}, Automatic NAT Detection Status:\s+Remote end\s*%{DATA:is_remote_natted}\s*behind a NAT device\s+This\s+end\s*%{DATA:is_local_natted}\s*behind a NAT device
+# ASA-5-713257
+CISCOFW713257 Phase %{INT:phase} failure:  Mismatched attribute types for class %{DATA:class}:  Rcv'd: %{DATA:received_attribute}  Cfg'd: %{GREEDYDATA:configured_attribute}
```

Comment:
This pattern has a few double spaces, is this intended?

Reply:
Yes.
Here is a sample log message:
```
Phase 1 failure:  Mismatched attribute types for class Group Description:  Rcv'd: Group 2  Cfg'd: Group 5
```

Reply:
I see, my question came from doing a quick search online and finding references to that log entry with only 1 space: https://supportforums.cisco.com/t5/remote-access/ipsec-vpn-to-asa-5520/td-p/1528993

Maybe we can be safe here and replace these potential "more than one" spaces with [`%{SPACE}`](https://github.com/logstash-plugins/logstash-patterns-core/blob/master/patterns/grok-patterns#L15) ?

Reply:
We could use '\s+' (1 or more spaces) but not %{SPACE} because it is defined as '\s*' (zero or more spaces).
Using %{SPACE} could break the pattern.

Reply:
BTW, if it sheds more light on the subject, all the ASA-5-713257 messages I have ever seen (I have been parsing Cisco ASA logs for 3 years now) have always had the 2 spaces in those places.